### ISSUE TITLE
Nav bar on splash screen

### ISSUE THREAD

**Issue by KODerFunk** [mesmotronic/cordova-fullscreen-plugin#17]
Meizu m1 note phone on Android 4.4.4 (phonу with touch-home-button only)
cordova 5.4.1 / android platform 4.1.1
use cordova-plugin-fullscreen@1.0.2
I have nav-bar on splashscreen at start and web-content deformation after hide splashscreen.
No effect from plugin(

**Comment by rafaelbiten** [comment]
I was having the same 'deformation' problem on my splash screen.
Adding `<preference name="SplashMaintainAspectRatio"	value="true" />` to my config.xml file fixed the issue. See if that helps @KODerFunk

**Comment by KODerFunk** [comment]
Yep, i use `SplashMaintainAspectRatio`, but navbar **present**. And present when keyboard (`ionic-plugin-keyboard`) open (it worst, view box so small).
